src/Qir/Runtime/lib/Simulators/ToffoliSimulator.cpp

```diff
@@ -109,11 +110,24 @@ namespace Quantum
             return std::abs(actualZeroProbability - probabilityOfZero) < precision;
         }
 
+        // Deprecated, use `DumpMachine()` and `DumpRegister()` instead.
         void GetState(TGetStateCallback callback) override
         {
             throw std::logic_error("operation_not_supported");
         }
 
+        void DumpMachine(const void* location) override
+        {
+            std::cerr << __func__ << " is not yet implemented" << std::endl;
+            // TODO: How to implement: https://github.com/microsoft/qsharp-runtime/pull/634#discussion_r615183791
+        }
+
+        void DumpRegister(const void* location, const QirArray* qubits) override
+        {
+            std::cerr << __func__ << " is not yet implemented" << std::endl;
+            // TODO: How to implement: https://github.com/microsoft/qsharp-runtime/pull/634#discussion_r615183791

```

Comment:
Probably worth separating out into it's own issue specific to the Toffoli simulator. Then you can link the issue here, and link to the PR comment in the issue.

Reply:
Thanks for the hint!  #645. Changing the code to
```c++
        void DumpMachine(const void* location) override
        {
            std::cerr << __func__ << " is not yet implemented" << std::endl;    // #645
        }

        void DumpRegister(const void* location, const QirArray* qubits) override
        {
            std::cerr << __func__ << " is not yet implemented" << std::endl;    // #645
        }
```